src/query/storages/iceberg/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "common-storages-iceberg"
+version = { workspace = true }
+edition = "2021"
+authors = ["Databend Authors <opensource@datafuselabs.com>"]
+license = "Apache-2.0"
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+common-catalog = { path = "../../catalog" }
+common-exception = { path = "../../../common/exception" }
+common-expression = { path = "../../expression" }
+common-meta-app = { path = "../../../meta/app" }
+common-meta-types = { path = "../../../meta/types" }
+common-storage = { path = "../../../common/storage" }
+
+async-trait = "0.1"
+chrono = { workspace = true }
+futures = "0.3"
+iceberg-rs = { git = "https://github.com/clslaid/iceberg-rs", branch = "feat/v1/manifests" }
```

Comment:
Please move the fork crate to https://github.com/datafuse-extras

Reply:
@ClSlaid please visit @PsiACE for help.

Reply:
Transfer finished, will submit another PR to redirect.